scripts/check_frontend_coverage.py

```diff
@@ -63,6 +63,7 @@
     'capitalize.filter.ts',
     'capitalize.pipe.ts',
     'ChangeObjectFactory.ts',
+    'ckeditor.import.ts',
```

Comment:
I'm not sure if this file should be included in the whitelist, should we not include this type of file in frontend coverage reports? cc @nithusha21 @seanlip

Reply:
I think it's up to you to decide on what the correct conventions are (perhaps together with @bansalnitish). Just make sure that the rule you come up with is a general one, and not related to only the specific file.

(Also, please document it in this file and the appropriate wiki page. Thanks!)

Reply:
Hi @marianazangrossi, Could you please let me know why should we not include this type of file in coverage reports? I guess it's OK to keep this in coverage report.

Reply:
I think it is ok as it is a ts file after all... Though I leave the final decision to you @marianazangrossi!

Reply:
I was just confused because there is no logic in this file. But if it's being tracked by Karma, lets keep it in the whitelist.